apps/openmw/mwmechanics/character.cpp

```diff
@@ -2421,6 +2421,10 @@ void CharacterController::updateHeadTracking(float duration)
     if (!head)
         return;
 
+    // Disable head tracking for player in First person mode
+    if (mPtr == getPlayer() && MWBase::Environment::get().getWorld()->isFirstPerson())
+        return;
```

Comment:
Maybe add setting the head to non-rotated state here before return (I'm guessing with `mAnimation->setHeadPitch` and `mAnimation->setHeadYaw`). Because this only stops rotating the head, but if it's already rotated, e.g. switching to FP during combat, it will stay that way.

Reply:
You can check how I did for unconscious actors in the actors.cpp - I call the setHeadTrackTarget() with empty target.

Reply:
[Here](https://github.com/OpenMW/openmw/pull/1550/files), right?

Reply:
Yes.

Reply:
So should I move code to another place or not?

Reply:
I'm a little drunk now so... but I guess whichever is simpler that achieves what I said in the first comments, it's up to you.